apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/TextResponse/TextResponse.stories.tsx

```diff
@@ -26,24 +28,16 @@ const TextResponseStory: Meta<typeof TextResponse> = {
   }
 }
 
-type Story = StoryObj<
+const Template: StoryObj<
   ComponentProps<typeof TextResponse> & { block: TreeBlock<TextResponseBlock> }
->
-
-const Template: Story = {
+> = {
   render: ({ ...args }) => {
     return (
-      <Stack
-        direction="row"
-        spacing={4}
-        sx={{
-          overflowX: 'auto',
-          py: 5,
-          px: 6
-        }}
-      >
-        <TextResponse {...args.block} />
-      </Stack>
+      <EditorProvider initialState={{ selectedBlock: { ...args.block } }}>
+        <Drawer title="Feedback Properties">
+          <TextResponse {...args.block} />
```

Comment:
In where you pass the args in below, you need to extract out the properties from `block` into `args`. Eg
```
export const Default = {
    ...Template,
    args: {
        __typename: 'TextResponseBlock`,
        id: 'textResponseBlock.id',
        ....
```
 

```suggestion
          <TextResponse {...args } />
```